src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
 // write code here
+const headline = document.querySelector('thead');
+let lastClickedIndex = null;
+
+function getCleanedValue(row, index) {
+  const cell = row.querySelectorAll('td')[index];
+
+  if (!cell) {
+    return '';
+  }
+
+  const cleanedCell = cell.textContent.trim().replace(/[$,]/g, '');
+
+  return isNaN(cleanedCell) ? cleanedCell : parseInt(cleanedCell);
+}
```

Comment:
This function `getCleanedValue` is doing a lot of things: it's getting the cell, cleaning its content, and converting it to a number if possible. It might be a good idea to divide it into smaller functions, each with a single responsibility. This approach, known as the Single Responsibility Principle, makes your code easier to read and maintain.